testdata/scripts/seed.txt

```diff
@@ -0,0 +1,36 @@
+# Check the binary with a given base64 encoded seed
+garble -debugdir=./debug -literals -seed=OQg9kACEECQ= build main.go
+exec ./main
+cmp stdout main.stdout
+grep \\x42\\x49\\x29\\x2f\\x5a\\x13\\xab\\xef\\xfa\\x26\\x33\\x3e\\x46\\xa9\\xec\\x3e\\x96\\xa3\\x73\\x15\\x7c\\xa2\\x38\\xf2\\xf0\\xaf\\x58\\x6a\\xac\\x2b\\x50\\x94\\x3b\\x5b\\x3e\\xda\\x2f\\xe7 $WORK/debug/main/z0.go
+grep zWuVH6jac $WORK/debug/main/z0.go
+grep \\xea\\x20\\xd0\\xe8\\x66\\x9c\\x72\\x24\\x9a\\x94\\xec\\x6a\\x8f\\x82\\xd3\\xdc $WORK/debug/main/z0.go    
```

Comment:
I'm pretty sure you could simplify this a bit, like `'\xAB\xCD'`. With single quotes, you shouldn't need the double backslashes. And you sohuld capitalize the hex, to clearly separate it from the `\x` bits.

Reply:
Also, please document what all these bytes mean near the top of the file. For example, is this one of the previous strings in base16?

Reply:
Tested you need double backslashes even when using single quotes.

Reply:
Ah, too bad. Might still be good to add quotes just for the sake of being explicit that this is all a chunk of bytes.